liri.js

```diff
@@ -0,0 +1,207 @@
+				      name: "movie"
+				    }
+				])
+			.then(function(inquirerResponse) {
+				movie(inquirerResponse.movie);
+			})
+			break;
+
+			case '["do-what-it-says"]':
+			doIt();
+			break;
+
+			default:
+			console.log("Bye!")
+	}
+	});
+};
+
+function twitter(user){
+	var client = new Twitter ({
+		consumer_key : keys.twitterKeys.consumer_key,
+		consumer_secret : keys.twitterKeys.consumer_secret,
+		access_token_key : keys.twitterKeys.access_token_key,
+		access_token_secret : keys.twitterKeys.access_token_secret,
+	});
+	var params = {
+		screen_name: user,
+		count: 20};
+	client.get('statuses/user_timeline', params, function(error, tweets, response) {
+	  if (!error) {
+	  	for (var i = tweets.length - 1; i >= 0; i--) {
+	  		log((i+1)+") "+tweets[i].created_at.substring(0,19)+ " "+ tweets[i].text);
+	  	}
+	  }
+	  else log(error);
+	});
+};
+
+function spotify(song) {
+	log("song: "+ song);
+	var spotify = new Spotify({
+	  id: keys.spotifyKeys.client_id,
+	  secret: keys.spotifyKeys.client_secret
+	});
+	if(song == "") { 
+		song = "The Sign";
+	}
+
+	spotify.search({ type: 'track', query: song }, function(err, data) {
+	  if (err) {
+	    return log('Error occurred: ' + err);
+	  };
+	  for (var i = 0; i < data.tracks.items.length; i++) {
+	  	log("============= "+(i+1)+" ==========================");
+	  	log("                                      ");
+	  	log("Artist: "+data.tracks.items[i].artists[0].name);
+	  	log("Song: "+data.tracks.items[i].name);
+	  	if(data.tracks.items[i].preview_url){
+	  		log("Preview: "+data.tracks.items[i].preview_url);
+	  	};
+	  	log("Album: "+data.tracks.items[i].album.name);
+	  	log("                                      ");
+	  }
+	});
+};
+
+function movie(movieName) {
+	if(movieName == ""){
+		movieName = 'Mr. Nobody.';
+	}
+		var queryUrl = "http://www.omdbapi.com/?t=" + movieName + "&y=&plot=short&apikey=40e9cece";
+		request(queryUrl, function(error, response, body) {
+			if (!error && response.statusCode === 200) {
+				if(JSON.parse(body).Title){
+			  		log("Title: " + JSON.parse(body).Title);
+				}
+				else log(movieName+ "was not found in the OMDB Database");
+			  	if(JSON.parse(body).Year){
+			  		log("Year: " + JSON.parse(body).Year);
+			  	}
+			  	if(JSON.parse(body).Ratings) {
+				  	if(JSON.parse(body).Ratings[0]){
+				  		log("Ratings: " + JSON.parse(body).Ratings[0].Source + " " + JSON.parse(body).Ratings[0].Value);
+				  	}
+				  	if(JSON.parse(body).Ratings[1]){
+				  		log("Ratings: " + JSON.parse(body).Ratings[1].Source + " " + JSON.parse(body).Ratings[1].Value);
+				  	}
+				  }
+			  	if(JSON.parse(body).Country){
+			  		log("Country: " + JSON.parse(body).Country);
+			  	}
+			  	if(JSON.parse(body).Language){
+			  		log("Language: " + JSON.parse(body).Language);
+			  	}
+			  	if(JSON.parse(body).Plot){
+			  		log("Plot: " + JSON.parse(body).Plot);
+			  	}
+			  	if(JSON.parse(body).Actors){
+			  		log("Actors " + JSON.parse(body).Actors);
+			  	}
```

Comment:
This is messy, you can just destructure it like so:
```javascript
let {Title, Year, Ratings, Country, Language, Plot, Actors} = JSON.parse(body)
log("some message containing the data with \n breaks")
```
it is important to try to call functions like log as little as possible since it is async it can have unpredictable behavior.